.env.dist

```diff
@@ -1,6 +1,6 @@
 DOCKER_PHP_IMAGE=8.1
 DOCKER_MYSQL_IMAGE=8.0
-DOCKER_ELASTICSEARCH_IMAGE=7.17
+DOCKER_ELASTICSEARCH_IMAGE=7.6
```

Comment:
You probably want to update this line

Reply:
?

Reply:
You changed it last week https://github.com/EmakinaFR/docker-magento2/pull/85/files to keep up-to-date with Magento, 7.6 is not supported anymore in the recent versions of Magento

Reply:
Oh yes. I didn't understand why you said "want" instead of "don't want" in your previous comment :)